src/main/scala/chisel3/experimental/inlinetest/InlineTest.scala

```diff
@@ -95,6 +95,14 @@ object TestHarnessGenerator {
   */
 trait HasTests[M <: RawModule] { module: M =>
 
+  /** Whether inline tests will be elaborated as a top-level definition to the circuit. */
+  protected def elaborateTests: Boolean = true
+
+  private val builderContext = internal.Builder.captureContext()
+
+  private def shouldElaborateTest(testName: String) =
+    builderContext.inlineTestIncluder.shouldElaborateTest(module.desiredName, testName)
```

Comment:
```suggestion
  private val inlineTestIncluder = internal.Builder.captureContext().inlineTestIncluder

  private def shouldElaborateTest(testName: String) =
    inlineTestIncluder.shouldElaborateTest(module.desiredName, testName)
```
Captring the entire Builder context here feels a bit extreme, I think we should just grab the test includer.